src/modules/universal-discovery/components/bookmarks/bookmarks.component.js

```diff
@@ -41,7 +36,6 @@ export default class BookmarksComponent extends Component {
                 perPage={bookmarksPerPage}
                 contentTypesMap={contentTypesMap}
                 title={tableTitle}
-                noItemsMessage={noBookmarksMessage}
```

Comment:
Why bookmark tables will not have this message anymore?

Reply:
It was never used in Bookmarks, it must have stayed after some modifications in the past.